dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/OracleConnectorScope.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2022-2023 Google LLC
+ * Copyright 2013-2021 CompilerWorks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.edwmigration.dumper.application.dumper.connector.oracle;
+
+import com.google.edwmigration.dumper.application.dumper.connector.LogsConnector;
+import com.google.edwmigration.dumper.plugin.lib.dumper.spi.OracleLogsDumpFormat;
+import com.google.edwmigration.dumper.plugin.lib.dumper.spi.OracleMetadataDumpFormat;
+
+enum OracleConnectorScope {
+  LOGS,
+  METADATA,
+  STATS;
+
+  String toDisplayName() {
+    if (this == METADATA) {
+      return "oracle";
+    } else {
+      return "oracle-" + getResultType();
+    }
+  }
+
+  String toFileName(boolean isAssessment) {
+    String timeSuffix = getTimeSuffix(isAssessment);
+    return String.format("dwh-migration-oracle-%s%s.zip", getResultType(), timeSuffix);
+  }
+
+  String toFormat() {
+    switch (this) {
+      case LOGS:
+        return OracleLogsDumpFormat.FORMAT_NAME;
+      case METADATA:
+        return OracleMetadataDumpFormat.FORMAT_NAME;
+      case STATS:
+        return "oracle.stats.zip";
```

Comment:
This looks like it belongs in `OracleLiteDumpFormat` and checked similar to the other cases above?

I'm referencing OracleLiteDumpFormat from the "Oracle Lite data extraction" Google doc.

Reply:
You're right about the discrepancy with the doc, thanks for spotting it. Some later offline discussions aren't reflected in the doc. I’ll fill in the missing information in the design and let you know so that we can continue the discussion.

Reply:
The doc and discussion were updated by me and @kpiotrowski, `OracleLiteDumpFormat` is now optional.

The main purpose of adding OracleLiteDumpFormat in Dumper SPI would be to provide the Translation team with a format that they can use to read Dumper outputs. Outputs of this connector won’t be used by Translation, so we can omit this format in the shared definitions.

Reply:
Ok. I still think it would be a good idea to have the format class for consistency, if nothing else. But I leave this one up to you.

Reply:
I'd prefer not to add this. I agree that adding it would be more consistent with other connectors, but it would also add clutter to the SPI which is used externally.